doc/base/classes.xml

```diff
@@ -49453,46 +49475,77 @@
 	</methods>
 	<members>
 		<member name="anti_aliasing" type="bool" setter="set_anti_aliased" getter="is_anti_aliased" brief="">
+			Anti Aliasing draws a small ring around edges. This ring fades to transparent.
+			As a result edges look much smoother. This is only noticable when using rounded corners.
 		</member>
 		<member name="anti_aliasing_size" type="int" setter="set_aa_size" getter="get_aa_size" brief="">
+			This changes the size of the faded ring. Higher values can be used to achieve a "blurry" effect.
 		</member>
 		<member name="bg_color" type="Color" setter="set_bg_color" getter="get_bg_color" brief="">
+			The background color of the stylebox.
 		</member>
 		<member name="border_blend" type="bool" setter="set_border_blend" getter="get_border_blend" brief="">
+			When set to true, the border will fade into the background color.
 		</member>
 		<member name="border_color" type="Color" setter="set_border_color" getter="get_border_color" brief="">
+			Sets the color of the border.
 		</member>
 		<member name="border_width_bottom" type="int" setter="set_border_width" getter="get_border_width" brief="">
+			Border width for the bottom border.
 		</member>
 		<member name="border_width_left" type="int" setter="set_border_width" getter="get_border_width" brief="">
+			Border width for the left border.
 		</member>
 		<member name="border_width_right" type="int" setter="set_border_width" getter="get_border_width" brief="">
+			Border width for the right border.
 		</member>
 		<member name="border_width_top" type="int" setter="set_border_width" getter="get_border_width" brief="">
+			Border width for the top border.
 		</member>
 		<member name="corner_detail" type="int" setter="set_corner_detail" getter="get_corner_detail" brief="">
+			This sets the amount of vertices used for each corner. Higher values result in rounder corners but take more processing power to compute.
+			By choosing in value you should take the corner radius ([method set_corner_radius]) into account.
+			For corner radius smaller than 10: 4-5 should be enough
+			For corner radius smaller than 30: 8-12 should be enough
+			
```

Comment:
Same issue here, this line and the new ones could be removed, they'll just make two empty paragraphs for no gain.

Reply:
changed to use much less lines